GUI/CalibrateTaskWindow.xaml.cs

```diff
@@ -270,5 +272,50 @@ private void KeyPressed(object sender, KeyEventArgs e)
                 CancelButton_Click(sender, e);
             }
         }
+
+        internal SearchModifications SearchMod = new SearchModifications();
```

Comment:
change `FilterTree` into a static method in `SearchModifications` class; no need to initialize a `SearchModifications` object

Reply:
so my idea behind SearchModifications is to reduce code redundancy for each **window** (CalibrateTask, SearchTask, XLSearchTask, ...) - so each instance of SM would set the timer, track which search box is used (fixed/var), and filter the mods. what would be the alternative?

Reply:
I like the idea of keeping one.

Reply:
what you described is fine and is a good idea (centralizing and not duplicating code). I just want it to be a static method